BLSQ/iaso
# POLIO-1762 POLIO-1764 POLIO-1765: Campaign combination cVDPV1 & cVDPV2 w/ 2 vaccines nOPV2 & bOPV

We will have to create a new Virus Type: cVDPV1 & cVDPV2 with the Vaccine Type: nOPV2 & bOPV associated

Related JIRA tickets POLIO-1762

## Self proofreading checklist




## Changes

- added new constants
- implement new hashed colors as shape color

## How to test

- Open scope for a campaign, you should have nOPV2 & bOPV option in the vaccines.
- Create a campaign, try to select  cVDPV1 & cVDPV2 as virus.
- 

If a specific config is required explain it here: dataset, account, profile, etc.

## Print screen / video

![Screenshot 2024-12-17 at 15 26 13](https://github.com/user-attachments/assets/7a507dca-37fb-4b4e-aa78-d0be49b60e08)


## Follow the Conventional Commits specification

The **merge message** of a pull request must follow the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) specification.

This convention helps to automatically generate release notes.

Use lowercase for consistency.

[Example](https://github.com/BLSQ/iaso/commit/8b8d7d3064138c1e57878f17b4eb922516ab0112):

```
fix: empty instance pop up

Refs: IA-3665
```

Note that the Jira reference is preceded by a _line break_.

Both the line break and the Jira reference are entered in the _Add an optional extended description…_ field.
